pkg/assets/assets_generate.go

```diff
@@ -16,19 +16,36 @@
 package main
 
 import (
-	"log"
+	"net/http"
+	"time"
+
+	// External dependencies should also be added to assets.go file.
+	"github.com/prometheus/alertmanager/pkg/modtimevfs"
+	"github.com/shurcooL/httpfs/union"
+	"github.com/shurcooL/vfsgen"
 
 	"github.com/kinvolk/lokomotive/pkg/assets"
 )
 
 func main() {
-	dirs := map[string]string{
-		assets.TerraformModulesSource: "../../assets/terraform-modules",
-		assets.ControlPlaneSource:     "../../assets/charts/control-plane",
-		assets.ComponentsSource:       "../../assets/charts/components",
+	directoriesToEmbed := map[string]http.FileSystem{
+		assets.TerraformModulesSource: http.Dir("../../assets/terraform-modules"),
```

Comment:
@invidian can this https://github.com/kinvolk/lokomotive/issues/817 be also done as part of this PR?

Reply:
I'd do it separately to reduce the scope of this PR. This is an cleanup PR, not enhancement one.